config/v1/types_authentication.go

```diff
@@ -43,13 +43,25 @@ type AuthenticationSpec struct {
 	// +optional
 	OAuthMetadata ConfigMapNameReference `json:"oauthMetadata"`
 
-	// webhookTokenAuthenticators configures remote token reviewers.
-	// These remote authentication webhooks can be used to verify bearer tokens
+	// webhookTokenAuthenticators is DEPRECATED, setting it has no effect.
+	WebhookTokenAuthenticators []WebhookTokenAuthenticator `json:"webhookTokenAuthenticators,omitempty"`
+
+	// webhookTokenAuthenticator configures a remote token reviewer.
+	// This remote authentication webhook can be used to verify bearer tokens
 	// via the tokenreviews.authentication.k8s.io REST API.  This is required to
 	// honor bearer tokens that are provisioned by an external authentication service.
-	// The namespace for these secrets is openshift-config.
+	//
+	// The namespace for the referenced secret is openshift-config. It should
+	// contain kube config file data which describes how to access the remote
+	// webhook service. For further details, see:
+	//     https://kubernetes.io/docs/reference/access-authn-authz/authentication/#webhook-token-authentication
+	//
+	// The key "kubeConfig" is used to locate the data within the referenced secret.
+	// If the secret or expected key is not found, the webhook is not honored.
+	// If the specified kube config data is not valid, the webhook is not honored.
+	// The namespace for this secret is determined by the point of use.
```

Comment:
How is the statement that the namespace is point-of-use related to the statement on line 54 that indicates the referenced secret's namespace is `openshift-config`?

Reply:
looks like that originates from a copy-paste error, that I copy-pasted. Fixed.